packages/host/tests/helpers/adapter.ts

```diff
@@ -183,10 +189,6 @@ export class TestRealmAdapter implements RealmAdapter {
       if (typeof value === 'string') {
         fileRefContent = value;
       } else {
-        let moduleURLString = `${this.#paths.url}${path.replace(/\.gts$/, '')}`;
-
-        this.#loader.shimModule(moduleURLString, value as object);
-
```

Comment:
so to be clear--the benefit of this PR is that it removes "magic" functionality where we were shimming modules for you behind the scenes, and rather, shimming modules should be an explicit thing that the developer does?

Reply:
My hope is that shimming modules is not something we'll need to do on loaders eventually. We can do it for packages using the virtual network handler, and for test realms using the test realm adapter.